source/reference/mongosqld.txt

```diff
@@ -718,6 +718,17 @@ These configuration file options are also available as system variables.
 For more information, see
 :ref:`System Variables <system-environment-variables>`.
 
+Load External Configuration Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You may use expansion directives in your configuration file to load 
+externally sourced configuration values. Expansion directives help 
+obscure confidential information like security certificates and 
+passwords.
+
+To learn more about using expansion directives, see :manual:`Externally Sourced Configuration File Values </reference/expansion-directives/>` 
+in the MongoDB manual.
+
```

Comment:
Besides this brief section, I considered a note at the very top of the Configuration File section, along the lines of:

.. note::
   You may store sensitive values outside of your configuration file 
   and load them at startup time with an automatic |api| request or 
   shell command. To learn more, see :manual:`Externally Sourced Configuration File Values </reference/expansion- 
   directives/>` in the MongoDB manual.
   
Let me know if you think that's a better way to include this info.

Reply:
Perhaps put the content in an include file, and include the content twice using the include file?

Reply:
Good idea, done